extensions/2.0/Vendor/EXT_structural_metadata/README.md

```diff
@@ -75,7 +74,7 @@ A schema may be embedded in the extension directly or referenced externally with
 >   "extensions": {
 >     "EXT_structural_metadata": {
 >       "schema": {
->         "id": "schema-001",
+>         "id": "schema_001",
```

Comment:
The id `schema-001` is invalid according to https://github.com/CesiumGS/3d-tiles/tree/draft-1.1/specification/Metadata#identifiers

Reply:
Now that you mention this: The current form does not allow the "usual representation" of a https://de.wikipedia.org/wiki/Globally_Unique_Identifier - should `-` be allowed?

Reply:
Hm good point. Nor would it support guids with a number as the first character. This could be problematic.

The reason for the limited character set is to allow IDs to be valid identifiers in GLSL, C, C++. If we support hyphens we might as well support full unicode. Engines that generate code based on the IDs - like custom shaders in CesiumJS - would need a translation step. We have something like that already for unicode batch table properties in CesiumJS: https://github.com/CesiumGS/cesium/pull/8785.

Should we drop the alphanumeric requirement for IDs?

Reply:
@javagl maybe we should move this discussion to a new issue so that it doesn't hold up this PR?